### DISCUSSION TITLE
Downgrading dotnet aspire

### DISCUSSION THREAD

**Discussion by anuraj** [microsoft/aspire#2829] [Q&A] [answered]
I used Aspire Daily build to explore Mongo Express feature. Now I am trying to install the version 4, I am getting the following error.

![image](https://github.com/dotnet/aspire/assets/276411/1962289a-6eac-43eb-b265-5cb5bcb5607b)

@davidfowl Can you help?

**Comment by nickna** [comment]
It looks like you added the package source for [early builds](https://github.com/dotnet/aspire/blob/main/docs/using-latest-daily.md), but later removed it, which led to a condition where Nuget can't find this build.

I would suggest re-adding the early builds nuget source and try uninstalling the aspire workload again.

If that doesn't work, there's the nuclear option:
Physically remove the Preview 5 installer from `%USERPROFILE%\.dotnet\workloads\aspire`
Run `dotnet nuget locals all --clear`
Run `dotnet workload list`

**Comment by davidfowl** [comment]
cc @joperezr

**Comment by anuraj** [comment, marked as answer]
It didn't worked for me. I don't have a `workloads` folder under `%USERPROFILE%\.dotnet`. I had to rollback the installation using the following command - ` dotnet workload update --source https://api.nuget.org/v3/index.json --from-rollback-file .\aspire-rollback.txt`

And here is the `aspire-rollback.txt`

```
{
  "microsoft.net.sdk.aspire": "8.0.0-preview.4.24156.9/8.0.100"
}
```
